common/v2/features/DevTools/DevTools.tsx

```diff
@@ -139,24 +136,24 @@ const DevToolsToggle = () => {
       onClick={toggleDevTools}
       style={{
         position: 'fixed',
-        bottom: 0,
-        right: 0,
+        top: 0,
+        left: 0,
         zIndex: 99,
-        height: '5rem'
+        width: 112
```

Comment:
Do we usually handle width like this?

Reply:
Honestly, I don't know how we do.
I don't like `rem` since its value is relative to the parent and because you never know the actual size.
@vilsbole any advice on this ?

Reply:
imho `rem` is useful for responsive design when you care about scaling your components according to screen size. 
It this case, since it's just a dev tool and we don't have a responsive grid today, I would opt for the simplest solution. Whatever you think is best @FrederikBolding

Reply:
@vilsbole If you don't have a problem with it, I don't either. Especially since its a dev tool.